content/en/blog/2021-09-17-gitops-one-stop-shop-event.md

```diff
@@ -0,0 +1,37 @@
+---
+author: dholbach
+date: 2021-09-17 12:30:00+00:00
+title: GitOps One-Stop Shop Event
+description: >
+  Flux will be part of the GitOps Days community special event again. Learn from vendors who chose Flux for their GitOps offering: Amazon, D2IQ, Microsoft, Weaveworks.
+url: /blog/2021/09/gitops-one-stop-shop/
+images:
+- /img/2021-09-gitops-one-stop-shop-event.jpg
+---
+
+![](/img/2021-09-gitops-one-stop-shop-event.jpg)
+
+Flux is going to be center stage at the [GitOps One Stop Shop Event](https://gitopsdays.com) on Oct 20th, 2021.
+
+Our friends over at GitOps Days will be hosting a special community edition to invite vendors who have chosen to back their GitOps offering by Flux. We are very excited that the ecosystem around GitOps is growing and so many chose Flux to bring the goodness we believe in to their users and customers.
```

Comment:
Is this fair to say? Are these vendors all "cloud providers"?

```suggestion
Our friends over at GitOps Days will be hosting a special community edition to invite cloud providers who have chosen to back their GitOps offering by Flux. We are very excited that the ecosystem around GitOps is growing and so many chose Flux to bring the goodness we believe in to their users and customers.
```

Reply:
AFAICS not everyone who is presenting is a cloud provider.

Reply:
Yes, D2IQ is not a cloud provider so vendor seems to be a better, more general, fit for both, product and cloud vendors.